### PR TITLE
Use variable timeouts depending on the advert size rendered.

### PR DESCRIPTION
**Pull request by JonNorman**
Whilst most ads can be refreshed after 30 seconds, some need to wait longer, or not be refreshed at all (**outstream** and **fluid** respectively).

Here, **fluid** is used as a proxy for our Native advert formats.

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/dfp/on-slot-viewable.js [hunk 1]
```diff
@@ -5,11 +5,29 @@ import { Advert } from 'commercial/modules/dfp/Advert';
...
+    const refreshThresholdMsDefault = 30000;
+    const refreshThresholdMsVideo = 90000;
+
+    const fluidSize = '0,0';
+
+    const sizeString = advert.size && advert.size.toString();
+    const videoSizes = ['620,1', '620,350'];
+
+    if (sizeString === fluidSize) {
+        return;
```
Comment (LATaylor-guardian) [inline static/src/javascripts/projects/commercial/modules/dfp/on-slot-viewable.js]: What are you returning in this case? Does number default to zero?

**Comment by LATaylor-guardian** [comment]
👍

File: static/src/javascripts/projects/commercial/modules/dfp/on-slot-viewable.js (hunk 1 above)
Comment (JonNorman) [reply static/src/javascripts/projects/commercial/modules/dfp/on-slot-viewable.js]: Returning nothing in this case - the return type of `?number` indicates that it is a potentially undefined.

File: static/src/javascripts/projects/commercial/modules/dfp/on-slot-viewable.js [hunk 2]
```diff
@@ -5,11 +5,29 @@ import { Advert } from 'commercial/modules/dfp/Advert';
...
+
+    return refreshThresholdMsDefault;
+};
 
 export const onSlotViewable = (event: ImpressionViewableEvent): void => {
     const advert: ?Advert = getAdvertById(event.slot.getSlotElementId());
-    if (advert) {
+    const viewabilityThresholdMs = advert && getViewabilityThreshold(advert);
+
+    if (advert && viewabilityThresholdMs) {
```
Comment (JonNorman) [inline static/src/javascripts/projects/commercial/modules/dfp/on-slot-viewable.js]: @LATaylor-guardian which is why I check it here - if I didn't then `flow` would complain, but it also represents the logic for working out whether we should refresh the advert; if the threshold is undefined then we shouldn't refresh.

Comment (LATaylor-guardian) [reply static/src/javascripts/projects/commercial/modules/dfp/on-slot-viewable.js]: Ah sorry - missed the ?.

File: static/src/javascripts/projects/commercial/modules/dfp/on-slot-viewable.js (hunk 1 above)
Comment (LATaylor-guardian) [reply static/src/javascripts/projects/commercial/modules/dfp/on-slot-viewable.js]: Ah sorry - missed the ?

**Pull request merged**